### ISSUE TITLE
Dropped items don't get pushed out of a block if standing inside the northern or western edge

### ISSUE THREAD

**Issue by TheNewHEROBRINEX** [pmmp/PocketMine-MP#3929]
### Issue description

- Expected result: Dropped items get pushed out of a block even if standing on an edge.
- Actual result: Dropped items remain inside the block and do not fall.

### Steps to reproduce the issue
1. Build a pillar of blocks
2. Break one of the blocks
3. Drop an item on an edge of the block that is below the one you broke (I was able to reproduce this only on north and west directions)
4. Place back the block you broke in step 2
5. Notice that the item doesn't fall

### OS and versions

* PocketMine-MP: pmmp/PocketMine-MP@9e85ee4
* PHP: 7.3.19
* Server OS: macOS 10.14.6
* Game version: Irrelevant

### Plugins

- If you remove all plugins, does the issue still occur? Yes

### Crashdump, backtrace or other files

Images showing the issue: https://imgur.com/a/l7FQstv

**Comment by dktapps** [comment]
I don't think this is actually a bug. The code responsible for the pushing is working fine. It's just not doing what you expect it to.

The position of the item entity is outside of the block. It's already technically "pushed out" by that measure. It's just lodged where it is because the AABB is partly colliding with the block below it.

**Comment by TheNewHEROBRINEX** [comment]
I think that if the item entity is outside of the first block it should also be able to free fall without colliding with the blocks below since the two blocks are aligned one on top of each other.

**Comment by dktapps** [comment]
Sure. I'm just saying that the current code is doing exactly what it was intended to, despite the fact it's not exactly logical.

**Comment by ConnorHopper** [comment]
no
